tests/website/test_my_api.js

```diff
@@ -359,7 +359,7 @@ async function testMyApiConverse(auth) {
         }, {
             id: 4,
             type: 'text',
-            text: 'Sorry, I did not understand that. Can you rephrase it?',
+            text: 'Sorry, there seems to be a problem with the Genie service at the moment. Please try again later.',
```

Comment:
Hold on from making this change. The NLP service is having some issue (https://github.com/stanford-oval/genienlp/issues/174) and is not staying up, but this is not the right reply.